tests/cypress/e2e/spec.cy.ts

```diff
@@ -52,7 +61,9 @@ describe('e2e tests', () => {
     // TODO: remove created dashboard. Once done, pseuso-random dashboard name can be removed.
   });
 
-  it('time-usage panel by service (single host)', {}, () => {
+  it('time-usage panel by service (single host - CEE)', {}, () => {
```

Comment:
Maybe put that into another `describe`. Like `describe("RAW")` and the other one would be `describe("CEE")`.

Reply:
Done (see latest commit).